src/gui/viewer.cpp

```diff
@@ -573,6 +673,23 @@ void InfoWidget::setup_ui() {
 ImageViewWidget::ImageViewWidget(Document*, QWidget* pParent)
 	: QWidget(pParent) {
 	setMinimumSize(256, 256);
+
+
+	// Make the checkerboard pattern default image.
+	// This is done here rather than in paint for performance's sake, as the checkerboard will never change.
+	checkerboard = QImage(checkerboard_size, checkerboard_size, QImage::Format_RGB32);
+	QRgb val;
+	val = qRgb(55, 60, 65);
+	QRgb val2;
+	val2 = qRgb(60, 64, 68);
+
+	for ( int nRow = 0; nRow < checkerboard_size; ++nRow)
+		for ( int nCol = 0; nCol < checkerboard_size; ++nCol)
+			if (((nRow / 20 + nCol / 20) % 2) == 0)
+				checkerboard.setPixel(nRow, nCol, val);
+			else
+				checkerboard.setPixel(nRow, nCol, val2);
```

Comment:
nit: i know other parts of the code don't use braces but this is a lot of logic to go braceless with

also, the magic number 20 should be 16 or 32 instead so it divides `checkerboard_size` evenly, and it should become a constant like `checkerboard_size`